### PR TITLE
Add freespace time param

### PR DESCRIPTION
**Pull request by marrts**
Added iterative spline parameterization to freespace motions to create smooth trajectories and give values to the velocity and acceleration fields in the trajectory messages.

### CODE REVIEW

**Comment by jrgnicho** [comment]
@marrts can you rebase?

**Comment by marrts** [comment]
Okay, should be good for testing now

File: crs_motion_planning/src/path_planning_utils.cpp [hunk 1]
```diff
@@ -844,22 +865,28 @@ bool crsMotionPlanner::generateFreespacePlan(const tesseract_motion_planners::Jo
...
+    for (int i = 0; i < joint_trajectory.points.size(); ++i)
     {
-      RCLCPP_INFO(logger_, "MODIFYING TRAJECTORY TIME OUTPUT");
-      for (int i = 0; i < joint_trajectory.points.size(); ++i)
-      {
-        joint_trajectory.points[static_cast<size_t>(i)].time_from_start.sec = 0;
-        joint_trajectory.points[static_cast<size_t>(i)].time_from_start.nanosec = 1e8;
-      }
+      joint_trajectory.points[static_cast<size_t>(i)].time_from_start.sec = 0;
+      joint_trajectory.points[static_cast<size_t>(i)].time_from_start.nanosec = 1e8;
```
Comment (jrgnicho) [inline crs_motion_planning/src/path_planning_utils.cpp:881]: So each point has a  0.1 timestamp?

**Review by jrgnicho — APPROVED** [review]

File: crs_motion_planning/src/path_planning_utils.cpp (hunk 1 above)
Comment (marrts) [reply crs_motion_planning/src/path_planning_utils.cpp:881]: Yes, this is if you are using the old gazebo trajectory plugin, that way each step is 0.1 seconds apart and no real time parameterization is performed

**Pull request merged**